packages/core/src/state-management/repository.ts

```diff
@@ -226,6 +226,22 @@ export class Repository {
     const opts = { ...DEFAULT_LOAD_OPTS, ...loadOptions }
 
     const [state$, syncStatus] = await this.loadingQ.forStream(streamId).run(async () => {
+      if (process.env.CERAMIC_ENABLE_V4_MODE) {
```

Comment:
the comment about the MID indexing test relates to this code. we never take [this path](https://github.com/ceramicnetwork/js-ceramic/blob/3b54c616ea799bbedcf38ccca009ee654e19dce0/packages/core/src/state-management/repository.ts#L290) as we return early as we're always considering it already_synced. I assume it will be updated when received, and this is fine and the test will pass then, but just want to make sure.